README.md

```diff
@@ -4,9 +4,24 @@ This is my personal config in windows11 and ubuntu.
 
 ## 多平台都支持
 
-1. <https://github.com/BurntSushi/ripgrep>
+1. <https://github.com/BurntSushi/ripgrep> (ripgrep, 40k+ stars)
 2. <https://github.com/sunlei/awesome-tools>
 3. <https://github.com/GitAlias/gitalias>
```

Comment:
The format is different from other codes. Why not write the name and purpose of the project within double brackets?